base_layer/wallet/src/output_manager_service/storage/database.rs

```diff
@@ -519,6 +525,18 @@ where T: OutputManagerBackend + Clone + 'static
             .and_then(|inner_result| inner_result)
     }
 
+    pub async fn cancel_pending_transaction_below_block_height(
```

Comment:
This method name mismatches the one actually called, where it only cancels __at__ the block height. Is there something going to be added in future?

Reply:
Oh I actually moved from this to just `at_block_height` because I realized it should only relate to the current block_height and let the monitoring protocols handle the ones at lower blockheight. Will update this.